inference-engine/src/inference_engine/include/ie/ie_transformations.hpp

```diff
@@ -85,4 +85,12 @@ INFERENCE_ENGINE_API_CPP(void) LowLatency(InferenceEngine::CNNNetwork& network);
  */
 INFERENCE_ENGINE_API_CPP(void) lowLatency2(InferenceEngine::CNNNetwork& network, bool use_const_initializer = true);
 
+/**
+ * @brief The transformation replaces the provided pairs of Parameter and Results with ngraph Memory layers
+ * ReadValue and Assign
+ * @param network A network to apply LowLatency transformation
+ * @param in_out_names Parameter/Result node names to replace.
+ */
+INFERENCE_ENGINE_API_CPP(void)
+makeStateful(InferenceEngine::CNNNetwork& network, std::map<std::string, std::string>& in_out_names);
```

Comment:
I believe we don't need new API in current IE?

CC @ilyachur

Reply:
@itikhono Do we need to have this transformation in applications which are working on old API?

Reply:
>>Do we need to have this transformation in applications which are working on old API?
@nkogteva @dorloff maybe you can answer Ilya's question. Do we need CNN Network API for the transformation  or ngraph/MO/python way is enough?

Reply:
@itikhono We need this functionality to be available in MO to enable testing of customer topologies. So I suppose we don't need CNNNetwork API for it. @dorloff Please confirm.

Reply:
Discussed with @dorloff. The CNN interface is unnecessary, I will delete it.

Reply:
I confirm. Sorry for nor having responded sooner.